.github/workflows/lock.yml

```diff
@@ -17,4 +17,4 @@ jobs:
         with:
           github-token: ${{ github.token }}
           issue-lock-inactive-days: 90
-￼         pr-lock-inactive-days: 90
+          pr-lock-inactive-days: 90
```

Comment:
What's happening here? I literally don't see anything different lol. Is that non-printable unicode?